.env.example

```
@@ -1,27 +1,22 @@
-NEXT_PUBLIC_PIMLICO_BUNDLER_URL=
-NEXT_PUBLIC_PIMLICO_PAYMASTER_URL=
-
-# CONTRACTS
-NEXT_PUBLIC_APP_ENV="development" # development | production
-NEXT_PUBLIC_BASE_SLICE_CONTRACT=
-NEXT_PUBLIC_BASE_USDC_CONTRACT=
-NEXT_PUBLIC_BASE_SEPOLIA_SLICE_CONTRACT=""
-NEXT_PUBLIC_BASE_SEPOLIA_USDC_CONTRACT=""
+# Stellar / Soroban
+NEXT_PUBLIC_STELLAR_NETWORK=testnet
+NEXT_PUBLIC_STELLAR_NETWORK_PASSPHRASE=Test SDF Network ; September 2015
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)